trade-finance-manager-ui/server/controllers/login/login-sso/login.controller.get-login.test.ts

```diff
@@ -79,5 +105,9 @@ describe('controllers - login (sso)', () => {
         .calledWith({ successRedirect: '/' })
         .mockResolvedValueOnce({ authCodeUrl: mockAuthCodeUrl, authCodeUrlRequest: mockAuthCodeUrlRequest });
     }
+
+    function mockFailedGetAuthCodeUrl() {
```

Comment:
Can we remove this? it's only used once, I think it'd be clearer to just have this in the one test